app/main.py

```diff
@@ -1,8 +1,46 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int = 1,
+                 clean_mark: int = 1,
+                 brand: str = "") -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
```

Comment:
these values have specific range of correct values. Let's check if they are correct, if not -> set some default correct value